aws-connect

```diff
@@ -208,6 +220,55 @@ elif [ "${action}" == "tunnel" ]; then
     --parameters "{\"portNumber\":[\"22\"],\"localPortNumber\":[\"${port}\"]}" \
     --region "${aws_region}" \
     --profile "${aws_profile}"
+elif [ "${action}" == "document" ]; then
+
+  if [ -z "${github_token_location}" ]
+  then
+    echo "No github token"
+    document_parameters="parameters='${document_parameters}'"
+  else
+    document_parameters="githubTokenLocation='${github_token_location}',parameters='${document_parameters}'"
+    echo "Github token found"
+  fi
+
+  echo "Running ${document_name} on instance ${instance_id}"
+  
+  sh_command_id=$( aws ssm send-command \
+    --instance-ids ${instance_id} \
+    --document-name "${document_name}" \
+    --comment "Document run using aws-connect" \
+    --output text \
+    --parameters "${document_parameters}" \
+    --region ${aws_region} \
+    --profile ${aws_profile} \
+    --query "Command.CommandId") 
+
+  output=$(aws ssm list-command-invocations \
+    --command-id ${sh_command_id} \
+    --details)
+
+  status=$(echo $output | jq -r '.[]' | jq -r '.[]' | jq -r '.Status')
```

Comment:
Don't use jq here.  jq doesn't come standard on the mac.  Use the `--query` option to aws cli.

https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-output.html#cli-usage-output-filter